src/msw/settings.cpp

```diff
@@ -337,7 +337,7 @@ extern wxFont wxGetCCDefaultFont()
     // font which is also used for the icon titles and not the stock default
     // GUI font
     LOGFONT lf;
-    if ( ::SystemParametersInfo
+    if ( wxSystemParametersInfo
```

Comment:
We do need the `window` parameter here however, but we don't have it in the places from which `wxGetCCDefaultFont()` is called :-( This is a problem because we're definitely going to end up with a font of the wrong size.